app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -870,7 +870,7 @@ private fun assertSponsoredShortcutTitle(sponsoredShortcutTitle: String, positio
     assertTrue(
         mDevice.findObject(
             UiSelector()
-                .className("android.widget.FrameLayout")
+                .resourceId("$packageName:id/top_site_item")
```

Comment:
All good, but I'd create a single topSiteItem val and reuse it.

Reply:
Sure, sounds good.

Reply:
As per our private discussion I'll leave it as it is.